src/js/angular/import/controllers.js

```diff
@@ -715,9 +734,9 @@ importViewModule.controller('UploadCtrl', ['$scope', 'toastr', '$controller', '$
         $scope.settings.name = file.name;
         const data = UploadRestService.createUploadPayload(file, $scope.settings);
         const uploader = startImport ? UploadRestService.uploadUserDataFile : UploadRestService.updateUserDataFile;
-        uploader($repositories.getActiveRepository(), file, data).success(function () {
+        uploader($repositories.getActiveRepository(), file, data).then(() => {
             $scope.updateList();
-        }).error(function (data) {
+        }).catch((data) => {
```

Comment:
When mixing `success/error` and `then/catch` keep in mind that the first returns only the body and the latter returns the whole respones object.

Reply:
`getError(data)` seems to handle this